.environment/gitleaks/gitleaks-config.toml

```diff
@@ -244,6 +248,7 @@ title = "PRIME ReportStream Gitleaks Configuration"
             '([0-1])\.0\.0\.0',
             '\d+\.\d+\.\d+\.\d+\.',                     # Fix for bug in regex (allows trailing dots)
             '1\.1\.1\.1',                               # Public DNS
+            '192\.(\d+\.){2}\d+',                        # Cheapo 10.0.0.0/8 range
```

Comment:
1. Comment is out of sync
2. check out 3f52c6483 (a commit by @ronaldheft-usds) which should address this; do a merge from `master` -> `gitleaks-issues` and eliminate this/your change to see if it fixes what you were seeing.

Reply:
Done

Reply:
See line 256, we have coverage for this (which is why I wanted that merged in :) ); this line can go now